apiRoutes/team_checklist.js

```diff
@@ -0,0 +1,118 @@
+const express = require("express")
+const router = express.Router()
+
+router.use(express.json())
+
+const { team_todolist } = require("../models")
+const { team_checklist } = require("../models")
+const { user } = require("../models")
+const { team } = require("../models")
+
+router.get("/", async (req, res)=>{
+    let data = await team_checklist.findAll( {include:[team_todolist]}).catch((err)=>{
+        console.log(err)
+        req.sendStatus(400)
+    })
+    res.send(data)
+})
+
+router.get("/:id", async (req, res)=>{
+    let checklsitID = req.params.id
+    let data = await team_checklist.findOne( {where:{checklist_ID: checklsitID}}).catch((err)=>{
+        console.log(err)
+        req.sendStatus(400)
+    })
+    res.send(data)
+})
+
+router.post("/add", async (req, res)=>{
+    //Create data
+    let size = await team_checklist.count()
+    let data = await team_checklist.create({
+        checklist_ID: req.body.checklist_ID,
+        todolist_ID: req.body.todolist_ID,
+        userlist: req.body.userlist,
+        name: req.body.name,
+        state: req.body.state,
+        checklist_index: size,
+        expire_datetime: req.body.expire_datetime,
+        due_datetime: req.body.due_datetime,
+        checklist_check: req.body.checklist_check
+    }).catch((err)=>{
+        console.log(err)
+        res.sendStatus(400)
+    })
+
+    //Added association with assign value upon created
+    let userlist = req.body.userlist
+
+    //If userlist is empty
+    if(!userlist) return res.status(400).send("Empty userlist")
+    
+    let assign_trueUser = userlist.true
+    let assign_falseUser = userlist.false
```

Comment:
What is this?

Reply:
In case when user want to both create checklist and assign other users to it, So I take list of user that has/not has been assign at first and then add them in to association table.

Reply:
Rename variables of request body (`userlist.true` and` userlist.false`) to be more readability.
Try not to name variables be like reserved words like `true` or `false`.